src/cassandra/uuid.rs

```diff
@@ -104,6 +104,56 @@ impl Uuid {
     }
 }
 
+impl From<uuid::Uuid> for Uuid {
+    fn from(id: uuid::Uuid) -> Uuid {
+        // implementation taken from serializetion.hpp encode_uuid()
```

Comment:
Typo - `serializetion` should be `serialization`? And please indicate where this file can be found - presumably the C/C++ Cassandra driver?

Reply:
tnx